framework/core/src/Filter/ValidateFilterTrait.php

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
Could these have phpstan return docstrings?

Reply:
the methods return types are explicit though, what would we need docblock return types for 😬

Reply:
`array<int>`, `array<string>`, etc, to narrow it from generic arrays

Reply:
oh, 🤦🏼